src/static/schema/query.types.json

```diff
@@ -0,0 +1,978 @@
+{
+  "kind": "QueryTypeDefinitionList",
+  "apiVersion": "query.grafana.app/v0alpha1",
```

Comment:
Might not understand everything here, but this feels a bit limited if all plugins share the same apiVersion and colliding kinds?

Reply:
In this case the apiVersion defines how to read *this* JSON file.  The apiVersion for each datasource is independent.

This is the equivalent to the apiVersion on a CRD *file*, not the custom resource (where the apiVersion is defined inside the CRD!)

Reply:
the schema for this file is the same across all data sources.  If/when we need to make breaking changes to the `QueryTypeDefinitionList` then the apiVersion would update.

Reply:
Got it 👍